internal/gitlab/webhooks.go

```
@@ -0,0 +1,377 @@
+
+	note := commentWebhook.ObjectAttributes.Note
+
+	// if the comment is a bot mention, only then we retry
+	if strings.Contains(note, h.Cfg.RetryCommand) {
+		h.OnRetryCommand(h.Client, w, &commentWebhook)
+	}
+
+}
+
+func (h *WebhookHandler) IncrementRetryCount(jobKey string) error {
+	_, err := h.Database.Exec("UPDATE running_jobs SET retry_count = retry_count + 1 WHERE key = ?", jobKey)
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)